base/doc/ltnews38.tex

```diff
@@ -278,67 +278,149 @@ \section{Hooks, sockets and plugs}
+%%FMi added
+The final argument contains the code that produces the current
+property value when when recording the value is requested.
+
+Then, to record the value of this new property,
+together with others, use this command.
 \begin{verbatim}
-\RecordProperties{mylabel}
-                 {chapter,page,currentlabel}
+  \RecordProperties{mylabel}
+                   {chapter,page,label}
 \end{verbatim}
-To reference the value the \cs{RefProperty} command is provided which takes 
-two arguments, the label and the property: 
+%%FMi added:
+This records the current values for the properties \texttt{chapter},
+\texttt{page}, and \texttt{label}, using \texttt{mylabel} as a label
+for the record.
+
+%%FMi changed: again following Lamport terminology
+To reference (i.e., to retrieve)
+this recorded value for use in a cross-reference to this chapter, use
+the \cs{RefProperty} command with two arguments: the label and the property.
 \begin{verbatim}
-\RefProperty{mylabel}{chapter}
+  \RefProperty{mylabel}{chapter}
 \end{verbatim}
- 
-The new module pre-declares a set of generally useful properties:
-\texttt{label} (the standard counter representation), \texttt{page}
-(the page representation), \texttt{title} (the title if set e.g.~by
-\pkg{nameref}), \texttt{target} (the name of the link target if set,
-e.g., by \pkg{hyperref}), \texttt{pagetarget} (the name of target
-added each page by \pkg{hyperref}), \texttt{pagenum} (the page as an
-arabic number), \texttt{abspage} (the absolute page number),
-\texttt{counter} (the name of the current counter), and \texttt{xpos}
-and \texttt{ypos} (the position on the page as set by the most recent
-\cs{pdfsavepos}: recording the properties should occur as soon after
-saving the position as possible).
-  
-The module provides both \LaTeXe{} camel case commands and
-\LaTeX3 programming layer commands.    
-
-For documentation see \texttt{texdoc ltproperties-doc}. 
+
+The
+%%FMi changed
+%extension itself contains
+\LaTeX{} kernel itself already contains
+declarations for some generally useful properties,
+including these:
+\begin{description}
+%%FMi changed:
+%\texttt{label} (the textual representation of the counter);
+\item[\texttt{label}] the textual representation of the \emph{current \textup{\cs{ref}} value}, see above;
+\item[\texttt{page}] the textual representation of the page;
+\item[\texttt{title}] the title, if set by, e.g.,~\pkg{nameref};
+\item[\texttt{target}] the logical name of the associated link target, if set by,
+e.g.,~\pkg{hyperref};
+\item[\texttt{pagetarget}] the logical name of the target added
+by \pkg{hyperref} at the origin of each shipped out page;
+\item[\texttt{pagenum}] the value of the \LaTeX{} counter \texttt{page} in Arabic numerals;
+%% NOT as an Arabic number -- Arabic numbers are "something totally different"!!
+%%FMi changed:
+%\texttt{abspage} (the value of the \texttt{shipout count});
+\item[\texttt{abspage}] the current number of shipped out pages;
```

Comment:
past  tense "shipped out" suggests that it starts at 0 but I think it starts at 1 so the value is 1 if you access this on the first page, even though It's really only set at shipout so perhaps.

`\item[\texttt{abspage}] the absolute page number of the page on which the label was recorded`

or something, not sure about that wording

Reply:
good point, but the property is independent of being recorded or not, so should not be describle in that way I think.

   the absolute page number of the page currently produced

maybe

Reply:
or just: the current number of shipped out pages plus 1;

Reply:
as it records `\g_shipout_readonly_int` you could perhaps point to ltshipout-doc.pdf?

Reply:
I wouldn't. The reference to that doc wouldn't help in this situation (the property is recorded at shipout) and the technical documentation there about \g_shipout_readonly_int would be rather confusing and we don't say that this is what apspage us looking at (and we shouldn't in ltnews). I think the above statement is fine.

Reply:
I rewrote it slightly

Reply:
Not sure that I like the “plus one”, better? longer! :

one more than the number of pages shipped out so far (thus it starts at 1 and is increased by 1 whenever a page us shipped out).

Reply:
Typo: us ==> is (near to end)